common/src/test/java/gov/cms/ab2d/common/util/DataSetup.java

```diff
@@ -186,9 +188,14 @@ public void deleteCoverageSearchEvent(CoverageSearchEvent event) {
         coverageSearchEventRepo.delete(event);
     }
 
-    public Contract setupContract(String contractNumber) {
+    public Contract setupContract(String contractNumber, OffsetDateTime attestedOn) {
```

Comment:
A much cleaner way would be to preserve the original api and add another overloaded method that takes in the specific date string.  

The body of the original method can be nothing more than
return setupContract(contractNumber, OffsetDateTime.now());